src/main/java/org/usfirst/frc4048/common/loggingv2/CommandLogger.java

```diff
@@ -46,4 +76,90 @@ public void log() {
             }
         }
     }
+
+    public void queueLogMessage(String key, byte[] value) {
```

Comment:
```suggestion
    public void logMessage(String key, byte[] value) {
```
The fact that this queues something is an implementation detail

Reply:
done